src/containers/AdminPersonList.jsx

```diff
@@ -26,10 +44,15 @@ const organizationFragment = `
     displayName
     email
     roles(organizationId: $organizationId)
+    todos(organizationId: $organizationId) {
```

Comment:
I am a little concerned about the performance implications for a large set of users.  It might be a good idea to limit this column and query by a checkbox, or to integrate it with pagination at least:
https://github.com/MoveOnOrg/Spoke/pull/996